docs/userdoc/AutoTuning.md

```diff
@@ -0,0 +1,46 @@
+# AutoTuning
+
+One of the primary features of AutoPas is the dynamic automated algorithm selection for the force calculation.
+Since AutoPas follows the philosophy of [Convention over Configuration](https://en.wikipedia.org/wiki/Convention_over_configuration), it will automatically tune itself to find the optimum of all its internal algorithm configurations.
+However, this naive tuning is just an exhaustive search by trialling all algorithms over successive iterations.
+While this is guaranteed to find the optimum, it is very costly and inefficient.
+Therefore, AutoPas offers a bunch of [Tuning Strategies](https://github.com/AutoPas/AutoPas/blob/master/src/autopas/options/TuningStrategyOption.h) that can be applied and combined for an optimal experience.
+
+## Tuning Loop
+To better understand what the strategies do, we first take a look at the way AutoPas does its tuning.
+
+The basic idea is that the AutoTuner class determines a configuration, which the LogicHandler then assembles and applies to the ParticleContainer.
```

Comment:
Adding links to important handlers / classes at the start of documentation might be useful for newcomers who would like to see their implementations :)

```suggestion
The basic idea is that the [AutoTuner](https://github.com/AutoPas/AutoPas/blob/master/src/autopas/tuning/AutoTuner.h) class determines a configuration, which the [LogicHandler](https://github.com/AutoPas/AutoPas/blob/master/src/autopas/LogicHandler.h) then assembles and applies to the [ParticleContainer](https://github.com/AutoPas/AutoPas/blob/master/src/autopas/containers/ParticleContainerInterface.h).
```

Reply:
We could also enhance other doc files with more links.